tools/python/transform/caffe_converter.py

```diff
@@ -559,13 +559,20 @@ def convert_crop(self, caffe_op):
         if param.HasField('axis'):
             axis_arg.i = param.axis
         axis_arg.i = 4 + axis_arg.i if axis_arg.i < 0 else axis_arg.i
-        axis =
+
+        offset_value = -1 * np.ones(4, dtype=np.int32)
+        offset_len = len(param.offset)
+        axis = axis_arg.i
+        if offset_len <= 1:
+            while axis < 4:
+                offset_value[axis] = 0 if offset_len < 1 else param.offset[0]
+                axis += 1
+        else:
+            offset_value[axis:] = param.offset
+
         offset_arg = op.arg.add()
         offset_arg.name = MaceKeyword.mace_offset_str
-        if len(param.offset) > 0:
-            offset_arg.ints.extend(list(param.offset))
-        else:
-            offset_arg.i = 0
+        offset_arg.ints.extend(offset_value)
 
     def convert_concat(self, caffe_op):
```

Comment:
I have no problems, but we need merge the two commits to a single commit

Reply:
Done